src/main/java/frc/robot/commands/AutoChargeStationBalance.java

```diff
@@ -0,0 +1,54 @@
+package frc.robot.commands;
+
+import com.ctre.phoenix.sensors.WPI_Pigeon2;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Calibrations;
+import frc.robot.subsystems.drive.DriveSubsystem;
+
+public class AutoChargeStationBalance extends CommandBase{
+    private final DriveSubsystem drive;
+    private final WPI_Pigeon2 gyro;
+    private final double NOT_MOVING_IN_Y = 0;
+    private final double NOT_ROTATING = 0;
+    private final boolean ROBOT_RELATIVE = false;
+
+
+    public AutoChargeStationBalance(DriveSubsystem drive){
+        this.drive = drive;
+        this.gyro = drive.getGyro();
+    }
+
+    // Called when the command is initially scheduled.
+    @Override
+    public void initialize() {}
+
+    // Called every time the scheduler runs while the command is scheduled.
+    @Override
+    public void execute() {
+        double pitchDeg = gyro.getPitch();
+
+        /** Velocity is [-1,1] */
+        double velocityFromAngleDeg = pitchDeg * Calibrations.CHARGE_STATION_PITCH_DEGREES_TO_VELOCITY;
+
+        double deadbandedVelocityFromAngleDeg = MathUtil.applyDeadband(velocityFromAngleDeg, Calibrations.CHARGE_STATION_DEADBAND_VELOCITY);
+
+        double matchTime = DriverStation.getMatchTime();
```

Comment:
This is `Time remaining in current match period (auto or teleop) in seconds`, right? Mind nothing that in a `//` comment right in here